src/lib/markdown-report-builder.ts

```diff
@@ -19,8 +19,14 @@ export default class MarkdownReportBuilder {
     return extensions
       .map(extension =>
         multiline(`
-        ## ${extension.displayName} \`${extension.id}\`
-        ${this.buildChangelog(extension)}`)
+        <details open>
+          <summary>
```

Comment:
I don't think those indentations are strictly needed, but I prefer to use it for HTML.